src/main/java/TaskList.java

```diff
@@ -0,0 +1,111 @@
+import java.util.List;
+import java.util.ArrayList;
+
+public class TaskList {
+    
+    private List<Task> list_of_Contents;
+
+    /**
+     * Constructor for TaskList object.
+     */
+    TaskList() {
+        list_of_Contents = new ArrayList<>();
+    }
+
+    /**
+     * Adds a task to taskList.
+     * @param task Task to be added.
+     */
+    public void addTask (Task task) {
+        list_of_Contents.add(task);
+    }
+
+    /**
+     * Prints the current task.
+     * @param task Task to be printed.
+     */
+    public void showNewContent(Task task) {
+        System.out.println("Got it. I've added this task: ");
+        System.out.println(task.toString());
+        System.out.println("Now you have "+ list_of_Contents.size() + " tasks in the list.");
+    }
+
+    /**
+     * Prints the whole taskList.
+     */
+    public void showAllContent() {
+            int no_of_tasks = list_of_Contents.size();
+
+            Ui ui = new Ui();
+            System.out.println("Here are the tasks in your list: ");
+            ui.printHorizontalLine();
+            if (no_of_tasks == 0) {
+                System.out.println("There is no tasks in the list, please add some tasks first ");
+            } else {
+                for (int i = 0; i < list_of_Contents.size(); i = i + 1) {
+                    String counter = Integer.toString(i + 1) + ". ";
+                    System.out.println(counter + list_of_Contents.get(i).timeConverted());
+                }
+            }
+            ui.printHorizontalLine();
+    }
+
+    /**
+     * Returns the number of task in the list.
+     * @return Integer indicating number of task in list.
+     */
+    public int getSizeOfList() {
+        return list_of_Contents.size();
+    }
+
+
+    /**
+     * Returns the taskList.
+     * @return TaskList.
+     */
+    public List<Task> getTheList() {
+        return list_of_Contents;
+    }
+
+    public void set_Task_As_Done(int n) {
+        System.out.println("Nice! I've marked this task as done: ");
+        Task task = list_of_Contents.get(n - 1);
+        task.set_Task_As_Done();
+        System.out.println(task.toString());
+    }
+
+    /**
+     * Deletes the nth task.
+     * @param n Index of the task to be deleted.
+     */
+    public void removeTask(int n) {
```

Comment:
n isnt a good name as it is not obvious what it represents, try using a name like numTask